### DISCUSSION TITLE
Installing frr and connecting it with core emulator

### DISCUSSION THREAD

**Discussion by Khubaib18** [coreemu/core#558] [General]
Hi, i have to install and integrate frr in core, but even after installing frr, the frr tab doesnt show inside the core routers, can anyone help me about this issue pls

**Comment by bharnden** [comment]
What version are you using?

The more recent installs attempt to help automate installing OSPF MDR, you would need to uninstall this as I believe FRR and Quagga would have conflicts at the same time.

There is no hard requirements for the FRR services, so if you are using a recent version you should see them available in the services dialog for a node.

**Reply by Khubaib18** [reply]
i have already removed quagga. i am using the most recent one as I only  issue the command to install it. but even after installing both frr and core, i cannt the the FRR service inside the router, i see quagga i dont know why

**Reply by bharnden** [reply]
While editing a session, you can right click a node to change the services being used. By default the "router" node will attempt to use OSPF MDR based services. You would have to manually change that for FRR or define a new node type that uses FRR services as you need.
